client/src/ZoomToggleButton.jsx

```diff
@@ -1,15 +1,9 @@
 import React from "react";
 import PropTypes from "prop-types";
 
-class ZoomToggleButton extends React.Component {
-    render() {
-        return (
-            <button onClick={this.props.paginator.toggleZoom}>
-                {this.props.paginator.state.zoom ? "Turn Zoom Off" : "Turn Zoom On"}
-            </button>
-        );
-    }
-}
+const ZoomToggleButton = ({ paginator }) => {
+    return <button onClick={paginator.toggleZoom}>{paginator.state.zoom ? "Turn Zoom Off" : "Turn Zoom On"}</button>;
```

Comment:
Extremely trivial point, but we could make the line shorter by factoring "Turn Zoom" out of the two strings and only making Off/On conditional.

Reply:
Made change